### DISCUSSION TITLE
x10 with Heyu Unable to get status of devices

### DISCUSSION THREAD

**Discussion by lukxz76** [HeyuX10Automation/heyu#59] [Q&A]
I've configured Heyu and all works fine without an essential command:
"heyu status Hx"
It's strange because in heyu monitor the 2-way module answer correctly but heyu does not receive the command correctly. I would like to schedule a "heyu status" in order to catch device changes operated directly using home switch that heyu does not received without that command.
I've tried using both v2.10.1 and v2.11-rc3 with the same issue

below the result of "heyu status A10" command

following "heyu monitor" result
05/20 22:28:11  sndc addr unit      10 : hu A10 (_no_alias_)
05/20 22:28:12  sndc func    StatusReq : hc A
05/20 22:28:13  rcvi addr unit      10 : hu A10 (_no_alias_)
05/20 22:28:13  rcvi func    StatusOff : hc A

at the same time, the "heyu status A10" command returns:
Unable to send command bytes

following output of "heyu -v status A10" commands:
pi@mercuryRPi:~ $ heyu -v status a10
Version:2.11-rc3
Searching for '/home/pi/.heyu/x10config'
Found configuration file '/home/pi/.heyu/x10config'
Heyu directory /home/pi/.heyu/ is writable.
Reading Heyu configuration file '/home/pi/.heyu/x10config'
lockpid: Checking for file '/var/lock/LCK..heyu.write.ttyUSB1'
Trying to lock (/var/lock/LCK..heyu.write.ttyUSB1)
lockpid: Checking for file '/var/lock/LCK..heyu.write.ttyUSB1'
/var/lock/LCK..heyu.write.ttyUSB1 is locked
Alert acks are 0x52, 0x12, 0x42
lockpid: Checking for file '/var/lock/LCK..heyu.relay.ttyUSB1'
lockpid: Checking for file '/var/lock/LCK..heyu.relay.ttyUSB1'
There was already a relay running (pid = 2619)
Sending address bytes: 04 6f

xwrite() called, count=2
xread() called, count=1, timeout = 1
xread() returning 1 byte(s). The first is 0x73
Checksum confirmed
xwrite() called, count=1
xread() called, count=1, timeout = 10
xread() returning 1 byte(s). The first is 0x55
Interface is ready.
Sending command bytes: 06 6f
xwrite() called, count=2
xread() called, count=1, timeout = 1
Alarm - timeout
xread() returning 0 bytes
Checksum NOT confirmed, no response from interface
xwrite() called, count=2
xread() called, count=1, timeout = 1
xread() returning 1 byte(s). The first is 0x75
Checksum confirmed
xwrite() called, count=1
xread() called, count=1, timeout = 10
xread() returning 1 byte(s). The first is 0x5a
send_buffer() interrupted by incoming command
Unable to send command bytes

lockpid: Checking for file '/var/lock/LCK..heyu.write.ttyUSB1'
munlock: Unlink file '/var/lock/LCK..heyu.write.ttyUSB1'

**Comment by jkrzyszt** [comment]
Answered on Yahoo group.

Unfortunately, Yahoo groups have been decommissioned, and I have no idea if any group archives exist.  Converting to QA discussion anyway.
